deck-o-cards.js

```diff
@@ -6,21 +6,29 @@ function deck_o_cards() {
 
   var cards = []; // deck
   var shuffledCards = []; // deck shuffled
-
+  var topCard = null;
```

Comment:
Instead of defining this here as null, I would just define and set below on line 25 at once.